apps/vs-code-react/src/app/export/workflowsSelection/advancedOptions.tsx

```diff
@@ -42,6 +42,14 @@ export const AdvancedOptions: React.FC = () => {
       defaultMessage: 'Generate infrastructure templates',
       description: 'Generate infrastructure templates',
     }),
+    INTEGRATION_ACCOUNT_SOURCE: intl.formatMessage({
+      defaultMessage: 'Make Integration Account the default source for Transformations and Schema validation',
```

Comment:
Consistent capitalization to use sentence case: https://learn.microsoft.com/en-us/style-guide/capitalization

```suggestion
      defaultMessage: 'Default to integration account as source for transformations and schema validation',
```